lib/auth/proto/auth.proto

```diff
@@ -236,6 +237,7 @@ service AuthService {
     // CreateResetPasswordToken creates ResetPasswordToken.
     rpc CreateResetPasswordToken(CreateResetPasswordTokenRequest)
         returns (services.ResetPasswordTokenV3);
-    // CreateUser inserts a new user entry to a backend.
-    rpc CreateUser(services.UserV2) returns (google.protobuf.Empty);
```

Comment:
Should we keep this for one release, marked as deprecated?
If auth server is updated but proxy, isn't all `CreateUser` calls will end up with errors.
`CreateUser` could just delegate to `UpsertUserV2` for now